app/components/footer/schema.yaml

```
@@ -1,8 +1,47 @@
 _description: |
-  footer
+  The footer at the bottom of all pages.
+_confirmRemoval: true
 
-footerLinks:
-  _label: Footer Links
+socialMediaNetworks:
+  _label: Social Media Links
+  _placeholder:
+    text: New Follow
+    height: 56px
+  _has:
+    input: complex-list
+    props:
+      -
+        prop: socialMediaNetwork
+        _label: socialMediaNetwork
+        _has:
+          input: radio
+          help: Select the network for this follow button
+          options: # new social media network should also be added to 'supportedSocialNetworks' in client.js
+                - name: 'Facebook'
+                  value: 'facebook'
+                - name: 'Twitter'
+                  value: 'twitter-black'
+                - name: 'Instagram'
+                  value: 'instagram'
+      -
+        prop: handle
+        _label: Social Media Handle
+        _has:
+            input: text
+            validate:
+              required: true
+      -
+        prop: icon
+        _label: Social Media Icon
+        _has:
+            input: text
+            help: Support SVG
+
+navegationLinks:
```

Remember to fix the typo for this property
```suggestion
navigationLinks:
```